packages/web-components/docs/components/ranking.md

```diff
@@ -37,12 +37,54 @@ Required:
 
 Optional:
 
+- `hideFooter` _boolean_
+  
+  Whether to hide the footer with download link.
+
+  Default: false
+
+- `highestTitle` _string_
+  
+  Chart title to show if `header` is not provided. Will only show if a
+  highest-to-lowest ranking is shown.
+
+- `lowestTitle` _string_
+
+  Chart title to show if `header` is not provided. Will only show if a
+  lowest-to-highest ranking is shown.
+
+- `rankingCount` _integer_
+  
+  How many places to show, e.g. the "N" in "Top-N".
+
+  Default: 5.
+
+- `showExploreMore` _boolean_
+
+  Whether to show "Explore more" link, which takes the user to Datacommons.org's visualization tools.
+
+- `showHighestLowest` _boolean_
+
+  Whether to show a high-to-low ranking stacked on top of a low-to-high ranking, in one chart.
```

Comment:
this actually just shows low to high with "..." between top N and bottom N
e.g., 1, 2, 3 ... 9, 10, 11

Reply:
Good catch! Updated comments and docs.